MDANSE/Tests/UnitTests/test_isotopes.py

```diff
@@ -0,0 +1,58 @@
+#    This file is part of MDANSE_GUI.
+#
+#    MDANSE_GUI is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+import numpy as np
+import pytest
+
+from MDANSE.Chemistry import ATOMS_DATABASE
+
+
+element_list = [atom_symbol for atom_symbol in ATOMS_DATABASE.atoms if all([char.isalpha() for char in atom_symbol])]
```

Comment:
```suggestion
element_list = [atom_symbol for atom_symbol in ATOMS_DATABASE.atoms if all([char.isalpha() for char in atom_symbol])]
```
`isalpha` on a string checks the whole string
```
>>> "hello".isalpha()
True
>>> "hello1".isalpha()
False
```